### PR TITLE
Issue #18848 adding leading slash uri

### PR DESCRIPTION
**Pull request by luisbacca**
This pull request add a leading slash to the uri parameter so the endpoint return the right folder structure.

### CODE REVIEW

**Comment by luisbacca** [comment]
https://github.com/dotCMS/core/issues/18848

File: dotCMS/src/main/java/com/dotcms/rest/api/v1/folder/FolderResource.java [hunk 1]
```diff
@@ -91,7 +91,11 @@ public final Response loadFolderByURI(@Context final HttpServletRequest httpServ
         final InitDataObject initData = this.webResource.init(null, httpServletRequest, httpServletResponse, true, null);
         final User user = initData.getUser();
         try{
-            Folder folder = folderHelper.loadFolderByURI(siteName,user,uri);
+            String uriParam = uri;
+            if(uriParam.charAt(0) != '/') {
```
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/rest/api/v1/folder/FolderResource.java:95]: Codacy found an issue: [Avoid using Literals in Conditional Statements](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5823758)

**Comment by erickgonzalez** [comment]
Is this a dupe of https://github.com/dotCMS/core/pull/18823 ??

**Comment by luisbacca** [comment]
> Is this a dupe of #18823 ??

Yes, resolved now.

**Review by jdotcms — APPROVED** [review]

**Review by erickgonzalez — APPROVED** [review]

**Review by freddyucv — CHANGES_REQUESTED** [review]
do it need a postman test?

**Review by dsilvam — APPROVED** [review]

**Comment by dsilvam** [comment]
Postman test for this will be included on a different branch

**Pull request merged**